File: easy/169/Solution.java
import java.util.Map;

class Solution {
    public int majorityElement(int[] nums) {
        Map<Integer, Integer> map = new HashMap<>();

        for(int i = 0; i < nums.length; i++){
            map.put(nums[i], map.getOrDefault(nums[i], 0) + 1);
        }
        int result = 0;
        int count = 0;
        for (Map.Entry<Integer, Integer> entry : map.entrySet()) {
            if(entry.getValue() > count){
                count = entry.getValue();
                result = entry.getKey();
            }
        }
        return result;
    }
}

File: easy/169/Solution.py
from collections import Counter

class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        dict = Counter(nums)
        result = 0
        count = 0
        for key, value in dict.items():
            if value > count:
                count = value
                result = key
        return result
    

    